db/model/model.js

```diff
@@ -1,33 +1,44 @@
-const db = require('../connection')
+const db = require("../connection");
 
 exports.fetchTopics = () => {
-    return db.query(`SELECT * FROM topics`).then(({
-        rows }) => {
-            return rows
-        })
-}
+  return db.query(`SELECT * FROM topics`).then(({ rows }) => {
+    return rows;
+  });
+};
 
 exports.fetchArcticlesById = (id) => {
-    return db.query(`SELECT * FROM articles WHERE article_id=$1`, [id]).then(({ rows }) => {
-        if (rows.length === 0) {
-            return Promise.reject({status: 400, msg: 'Id not found'})
-        }
-        return rows
-    })
-}
-
-exports.fetchUsers = () => {
   return db
-    .query(`SELECT * FROM users`)
+    .query(`ALTER TABLE articles ADD comment_count INT NOT NULL DEFAULT 0 `)
+    .then(() => {
+        return db.query(
+          `UPDATE articles SET comment_count=(SELECT COUNT(*)
+           FROM comments WHERE article_id=$1);`,[id]
+        );
+    })
+    .then(() => {
+      return db.query(`SELECT * FROM articles WHERE article_id=$1`, [id]);
+    })
     .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 400, msg: "Id not found" });
+      }
       return rows;
```

Comment:
We're only returning one thing from this endpoint so remove it from the array first!